src/bot.py

```diff
@@ -36,6 +36,11 @@ def int_conv(string: str) -> bool:
     i. e. `!nugget max` will transfer all nuggets to poster
     """
 
+    if db.check_comment(comment.id): # breaks completely when comments previously processed are reached
+        break
```

Comment:
Wouldn't that mean that the bot will exit the `for` loop and thus stop completely?

Reply:
Thought we were putting this in a `while` loop once finished, with `time.sleep()` statements after going through the cycle. If we only use `continue` the bot will keep going back to comments from months ago. 

The point of the `break` is if comments that were reached on the last cycle are reached, it goes into the `time.sleep()` again for however long we choose.

Reply:
Ah, right.